eureka/registry_test.go

```diff
@@ -212,6 +246,7 @@ var _ = Describe("Deregister", func() {
 							       },
 							       {
 								  "app":"APP-3",
+ 			                      "instanceId":"instance-1",
```

Comment:
Minor: same spaces/tabs issue on the line as mentioned above. Seems to happen on a few other lines as well (329, 339, 349, and 430).